compositor_render/src/transformations/text_renderer.rs

```diff
@@ -215,6 +215,11 @@ impl TextRendererCtx {
             swash_cache: Mutex::new(SwashCache::new()),
         }
     }
+
+    pub fn add_font(&self, source: Source) {
+        let mut font_system = self.font_system.lock().unwrap();
+        font_system.db_mut().load_font_source(source);
```

Comment:
what about errors?

Reply:
`load_font_source` doesn't return Result. In case of invalid font data it logs a warn to console.